examples/standalone/README.md

```diff
@@ -141,6 +141,20 @@ Install the following dependencies:
 - [Terraform](https://www.terraform.io/downloads.html) version 0.13.7 or later.
 - [curl](https://curl.haxx.se/) version 7.68.0 or later.
 
+### Cloud SDK configurations
+
+Acquire your user credentials to use for **Application Default Credentials**:
```

Comment:
I might suggest something to give a hint like the following, but cleaning the suggestion up (i.e don't need to use word for word)

Confirm you're using the correct user that can impersonate the service account used for deployment.  Use the following command: `gcloud auth list` to check which user is currently configured.

If you want to keep the current user, but want to use a different user credential for application default, use the following.

Reply:
We have two different types of login, the [gcloud auth application-default login](https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login) and the [gcloud auth login](https://cloud.google.com/sdk/gcloud/reference/auth/login).

The intention is guide to refresh the user that is logged in the `gcloud auth application-default login`, to avoid errors.

 The `gcloud auth list` shows the users that logged in the `gcloud auth login`.

Reply:
I like the updates, so closing this comment

Reply:
Just to add a few more details here. We did some additional tests and without using auth but passing the credentials we got the following error:
```

ERROR: (gcloud.builds.submit) You do not currently have an active account selected.
Please run:

  $ gcloud auth login

to obtain new credentials.

If you have already logged in with a different account:
    $ gcloud config set account ACCOUNT

to select an already authenticated account to use.
```

Only successful way was passing authentication and credentials. We also tested with only auth and without credentials which returned an error of "could not find default credentials".